source/includes/aggregation/vector-search.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$collection = $client->sample_mflix->embedded_movies;
+
+// start-basic-query
+$pipeline = new Pipeline(
+    Stage::vectorSearch(
+        index: 'vector_index',
+        path: 'plot_embedding',
+        queryVector: [-0.0016261312, -0.028070757, -0.011342932],
+        numCandidates: 150,
+        limit: 10,
+    )
```

Comment:
Our code style is to have trailing coma on EOL.
```suggestion
    ),
```